server/api/categories.js

```diff
@@ -32,8 +32,8 @@ router.get(`/all/parent`, async (req, res, next) => {
 
 router.get('/:categoryName/popular-paths', async (req,res,next) => {
   const category = req.params.categoryName
-  const query = `match(u:User)-[r:PATHS]->(p:Path)-[:CATEGORY]->(c:Category)
-      where c.name={category}
+  const query = `match(u:User)-[r:PATHS]->(p:Path {status: 'public'})-[:CATEGORY]->(c:Category)
+      where c.name={category} AND p.status = "public"
```

Comment:
Adding the `{status: 'public'}` to the node match and also in the `where` clause is redundant. I would pick one place or the other. Since you're already matching the category name in the `where` clause`, it might make sense just to keep all conditions there.